project1/index.html

```diff
@@ -0,0 +1,142 @@
+                pants that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly
+                at
+                home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+                doing
+                that
+                requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and
+                floppy hat
+                featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that
+                looks
+                like
+                this when something in my life has gone so terribly wrong.</p>
+
+            <p>Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+                New
+                York house with two willow trees I had bought with my fiancé.</p>
+
+            <p>Ten days later and I didn’t want to do anything I was supposed to do.</p>
+        </section>
+
+        <div>
+            &bull; &bull; &bull;
+        </div>
+
+        <blockquote>
+            <p>"When I looked at that mouse with her broom, I wondered which one of us was wrong
+                about who
+                <span><strong><em>I</em></strong></span> was."</p>
+            <!--I used a span around the 'I' because I wanted to add color to just the 'I' to add more emphasis because I don't think it was as noticeable that I was emphasizing it with just the <em> tag-->
+        </blockquote>
+
+        <div>
+            &bull; &bull; &bull;
+            <!--not sure if this is correct, but I wanted to put these as a separator of the sections and I am having trouble
+        making the spacing above and below to be equal-->
+        </div>
+
+        <section>
+            <p> I went to Texas to study the whooping crane because I was researching a novel. In my novel there were
+                biologists doing field research about birds and I had no idea what field research actually looked like
+                and
+                so the
+                scientists in my novel draft did things like shuffle around great stacks of papers and frown. The good
+                people of the Earthwatch organization assured me I was welcome on the trip and would get to participate
+                in
+                “real
+                science” during my time on the gulf. But as I waited to be picked up by my team in Corpus Christi, I was
+                nervous—I imagined everyone else would be a scientist or a birder and have daunting binoculars.</p>
+
+            <!--I don't like that it separated my text like this is there a way to fix it or clean it up?-->
+
+            <p>The biologist running the trip rolled up in in a large white van with a boat hitch and the words
+                <strong><em>BIOLOGICAL SCIENCES</em></strong> stenciled across the side. Jeff was forty-ish, and wore
```

Comment:
Same note about using both `strong` and `em` here. I wonder how you feel about `b` here instead? See the docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b